airgun/widgets.py

```diff
@@ -2229,3 +2229,54 @@ def items(self):
 
     def toggle(self, value):
         self.browser.click(self.ITEM.format(value))
+
+
+class InventoryBootstrapSwitch(Widget):
+    """Checkbox-like Switch control, representing On and Off state. But with
```

Comment:
This was written by mirek sometime back, I am adding it again because it's required for `insights_sync_switcher`.